controllers/component_build_controller_test.go

```diff
@@ -2016,6 +2029,88 @@ var _ = Describe("Component initial build controller", func() {
 			ensureNoPipelineRunsCreated(resourceKey)
 		})
 
+		It("should fail when no pipeline is selected based on predefined build pipeline selector", func() {
```

Comment:
Does it make sense to have a similar test for PaC?
Or maybe we should handle error cases for pipeline selection in the pipeline selector test for both? WDYT?